### PR TITLE
Increase Stake button, add max limits to each button, add directions/help text, fix LP balance

### PR DESCRIPTION
**Pull request by pakaplace**
DO NOT MERGE until testing the increase stake button.

**Stake Panel:**
- Withdraw <> Transfer unlocked LP tokens from Crucible to msg.sender
** Max amount = crucible balance - locked balance
- Increase stake <> Deposit more LP tokens to crucible lock
** Max amount = unlocked LP token balance
- Unstake and claim <> Claim/transfer existing rewards (proportional to LP token amount) from Aludel to recipient and unlock LP tokens from crucible
** Max amount= locked LP balance

**Mint Panel:**
- Mint and Stake <> Mint Crucible, Lock LP tokens, and Stake

I also added mist and ether rewards  to thee claim rewards modal.

### CODE REVIEW

File: src/Web3Context/index.tsx [hunk 1]
```diff
@@ -9,7 +9,10 @@ import {
   calculateMistRewards,
 } from "../contracts/aludel";
 import { getOwnedCrucibles } from "../contracts/getOwnedCrucibles";
-import { formatUnits } from "@ethersproject/units";
+import { getTokenBalances } from "../contracts/getTokenBalances";
+
+import { formatUnits, parseUnits } from "@ethersproject/units";
+import { AnyARecord } from "node:dns";
```
Comment (nuurdev) [inline src/Web3Context/index.tsx:15]: Imported by accident?

File: src/contracts/increaseStake.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+) {
+  const walletAddress = await signer.getAddress();
+
+  // Recipient doesn't make much sense when increasing stake
+  const args = {
+    crucible: crucibleAddress,
+    aludel: "0xf0D415189949d913264A454F57f4279ad66cB24d",
+    transmuter: "0xB772ce9f14FC7C7db0D4525aDb9349FBD7ce456a",
+    crucibleFactory: "0x54e0395CFB4f39beF66DBCd5bD93Cca4E9273D56",
+    recipient: walletAddress,
```
Comment (nuurdev) [inline src/contracts/increaseStake.ts:22]: Looks like recipient is not being used, should it be removed?

File: src/contracts/increaseStake.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+import transmuterAbi from "./transmuterAbi";
+
+export async function increaseStake(
+  signer: any,
+  crucibleAddress: string,
+  rawAmount: string
+) {
+  const walletAddress = await signer.getAddress();
+
+  // Recipient doesn't make much sense when increasing stake
```
Comment (nuurdev) [inline src/contracts/increaseStake.ts:16]: This can probably be removed

Comment (pakaplace) [reply src/contracts/increaseStake.ts:16]: >It's needed

File: src/contracts/increaseStake.ts (hunk 2 above)
Comment (pakaplace) [reply src/contracts/increaseStake.ts:22]: thanks

**Pull request merged**